phoebe/parameters/parameters.py

```diff
@@ -12024,7 +12024,8 @@ def _value(var, string_safe_arrays=False, use_distribution=None, needs_builtin=F
                     # to do from builtin import * (and even if I did, python
                     # yells at me for doing that), so instead we'll add them
                     # to the locals dictionary.
-                    locals()[func] = getattr(builtin, func)
+                    # See https://peps.python.org/pep-0667/
+                    sys._getframe().f_locals[func] = getattr(builtin, func)
```

Comment:
do you know perhaps which is the earliest version of python for which this will work?

Reply:
no, but it passes on the full CI matrix (3.8+).